build.sbt

```diff
@@ -212,33 +212,36 @@ lazy val scalameta = crossProject
 lazy val scalametaJVM = scalameta.jvm
 lazy val scalametaJS = scalameta.js
 
-lazy val scalahost = Project(id = "scalahost", base = file("scalahost"))
+lazy val scalahost = project
+  .in(file("scalahost/core"))
   .settings(
+    moduleName := "scalahost",
+    description := "Scala.meta semantic API integration for Scala 2.x (scalac).",
     publishableSettings,
     hasLargeIntegrationTests,
+    isFullCrossVersion,
+    libraryDependencies += "org.scala-lang" % "scala-compiler" % scalaVersion.value
+  )
+  .dependsOn(scalametaJVM)
```

Comment:
Scalafix will depend on this project to construct an offline mirror. See https://github.com/scalacenter/scalafix/issues/100 for why scalafix can't depend on scalahost-nsc directly.